app/proxy/envoy.yaml

```diff
@@ -114,7 +117,7 @@ static_resources:
       endpoints:
       - lb_endpoints:
         - endpoint:
-            address: { socket_address: { address: backend, port_value: 1751 }}
+            address: { socket_address: { address: 127.0.0.1, port_value: 1751 }}
```

Comment:
Frontend (after changing back to `localhost` URLs) doesn't connect to the backend; I think it's because of this line.

Reply:
There is a bug in envoy that requires this for kubernetes.
docker-compose.yaml has been edited for this to work in local deployment too. I will double check this isnt broken

Reply:
This does work when tested locally.
Backend may time out trying to connect while postgres starts up and is populated. You may need to run docker-compose a second time. Any help making backend wait for postgres is appreciated but its not a big problem for now to just run it again once postgres has been initialised.